projects/build-openssl.bat

```diff
@@ -44,7 +44,11 @@ rem ***************************************************************************
     set OS_PLATFORM=x86
   )
   if defined PROGRAMFILES(x86) (
-    set "PF=%PROGRAMFILES(x86)%"
+    if /i "%~1" == "vc14.3" (
```

Comment:
>Visual Studio used to be x86 only up to VS17

Ok then I suggest add a comment in both those files. Otherwise this PR looks fine.
```suggestion
    rem Visual Studio was x86 only prior to 14.3
    if /i "%~1" == "vc14.3" (
```